src/deletepy/models/config.py

```
@@ -0,0 +1,243 @@
+        Args:
+            total_items: Total number of items to process
+
+        Returns:
+            int: Optimal batch size
+        """
+        if total_items > self.large_dataset_threshold:
+            return 25
+        elif total_items > self.medium_dataset_threshold:
+            return 50
+        else:
+            return 100
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)